test/integration/default/verify/controls/aws_vpc_subnets.rb

```diff
@@ -0,0 +1,50 @@
+fixtures = {}
+[
+  'ec2_security_group_default_vpc_id',
+  'ec2_default_vpc_subnet_id',
+].each do |fixture_name|
+  fixtures[fixture_name] = attribute(
+    fixture_name,
+    default: "default.#{fixture_name}",
+    description: 'See ../build/ec2.tf',
+  )
+end
+
+control "aws_vpc_subnets client side filtering" do
```

Comment:
This resource has no server-side filtering, so there is no distinction between client and server-side filtering.  Puzzled at the control name.